### ISSUE TITLE
Pension Reactivation Email Content from Pension Benefits Team

### ISSUE THREAD

**Issue by vbellissimo** [department-of-veterans-affairs/va.gov-team#75872]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: Pension Benefits Team
- **OCTO product owner:** Emily Theis & Julie Strothman
- **Product name:** Pension Benefits Form / 527
- **Product manager:**  Vicky Bellissimo
- **Slack channel:** #benefits-pension 
- **Dedicated content writer on your team (if you have one):** Katherine Fung
- **Dedicated a11y specialist on your team (if you have one):** Gretchen Maciolek & Todd Rizzolo
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [ ] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [x] Something else: We'd like your support to create an email that will be sent to veterans who had an in-progress form at the time the online Pension form was deactivated in November 2023. Now that the form is being released to a certain percentage of traffic, we'd like to prepare an email that will let them know that the online form is again. [See launch release schedule in Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1704481084114/b171b0540054b2844cb7977dd7dd2dfb7aab9c48?wid=0-1704481087285&sender=u9d01ee38af217e4877e75660).

### What's the nature of your initiative and desired outcomes?

We'd like your support to create an email that will be sent to veterans who had an in-progress form at the time the online Pension form was deactivated in November 2023. Now that the form is being released to a certain percentage of traffic, we'd like to prepare an email that will let them know that the online form is again.

[See launch release schedule in Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1704481084114/b171b0540054b2844cb7977dd7dd2dfb7aab9c48?wid=0-1704481087285&sender=u9d01ee38af217e4877e75660).

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [x] Yes
- [ ] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

[See launch release schedule in Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1704481084114/b171b0540054b2844cb7977dd7dd2dfb7aab9c48?wid=0-1704481087285&sender=u9d01ee38af217e4877e75660).

We're aiming to have the email copy/template prepared by 2/16 so we can have it ready to send to veterans the following week.

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [x] None. This initiative isn’t going through the collaboration cycle.

We did a Staging Review for the re-launch of the form in January 2024 but this email template will not go through the Collaboration Cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** N/A

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.

For reference, here are some helpful artifacts to review for reqference:
- Past (Closed) ticket that has information about the email that was sent to veterans when the form was deactivated: #69310

## Next steps
- [ ] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).
```

```[tasklist]
### CAIA operations
- [x] Initial Review - @coforma-terry and @strelichl
```

```[tasklist]
### CAIA a11y tasks
```

```[tasklist]
### CAIA content tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/76153
```

```[tasklist]
### CAIA IA tasks
```

**Comment by katherine-fung** [comment]
Hi @vbellissimo @jstrothman, 

Here's a screenshot of the pension reactivation email for your review. I've drafted the copy in VA Notify: https://staging.notifications.va.gov/services/725bbdd4-5a6e-4688-8678-fae4323ef3a3/templates/aef34a57-0eb9-4df2-80df-206dac6d4e65

Please let me know if you have any feedback. Once you approve the final copy, engineering can go into the draft in VA Notify and adjust the links so they track correctly, and make any other adjustments that the team needs. 

Thanks! 

![Image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/105367381/8e241131-a497-430e-9126-d70722780c3b)

**Comment by vbellissimo** [comment]
@buddhima-coforma for your awareness.

**Comment by vbellissimo** [comment]
Thank you @katherine-fung !

**Comment by aplatt-coforma** [comment]
Approved by VA enablement crew
<img width="1274" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeTI5QXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--1425667cb0af5d425a2c6f46d9d321d64d6c43f8/Screenshot%202024-03-06%20at%207.13.21%E2%80%AFPM.png" alt="Screenshot 2024-03-06 at 7.13.21 PM.png" />

**Comment by danlim715** [comment]
I created a new template because I wasn't able to access the old one. Here's the updated link to the template: https://staging.notifications.va.gov/services/5e5cded3-3c76-46a3-9eef-cb63589e76be/templates/f7ec31d5-b39e-46c6-884c-ad4147a80e18. For those with VANotify access, please let me know if you aren't able to view this template.
